packages/p2p/src/components/toggle-container/toggle-container.scss

```diff
@@ -5,7 +5,7 @@
     border-radius: $BORDER_RADIUS;
 
     @include tablet-up {
-        margin: 2.4rem 0 4.8rem;
+        margin: 2.4rem 0;
```

Comment:
<img width="1264" alt="Screenshot 2023-05-25 at 9 39 19 AM" src="https://github.com/farrah-deriv/deriv-app/assets/103412909/287ccfae-c029-4326-8861-cf9f300a0e92">
<img width="1271" alt="Screenshot 2023-05-25 at 9 39 13 AM" src="https://github.com/farrah-deriv/deriv-app/assets/103412909/b9bd441a-dd1e-46cb-917e-acd1b65a0d59">

I just changed the bottom margin as it seemed different to production